bash_completions.go

```diff
@@ -103,6 +103,8 @@ __handle_reply()
     if [[ ${#COMPREPLY[@]} -eq 0 ]]; then
         declare -F __custom_func >/dev/null && __custom_func
     fi
+
+    __ltrim_colon_completions "$cur"
```

Comment:
I'm working on OSX.  `__ltrim_colon_completions` was added to bash-completion in [2009](http://anonscm.debian.org/cgit/bash-completion/bash-completion.git/commit/bash_completion?id=42302658705e3545471f2f4e53c2f758355a2bcc)